samples/drivers/led_ws2812/src/main.c

```diff
@@ -19,7 +19,7 @@
 /*
  * Number of RGB LEDs in the LED strip, adjust as needed.
  */
-#define STRIP_NUM_LEDS 12
+#define STRIP_NUM_LEDS 1
```

Comment:
Is this change related to the board code?  At min we need a comment in commit message about it.  Ideally it would be a separate commit.

Reply:
Oops, fixed.